web_app/backend/tests/conftest.py

```
@@ -0,0 +1,35 @@
+def mock_ai_council():
+    with patch("web_app.backend.main.AICouncil") as mock_class:
+        mock_instance = MagicMock()
+        mock_class.return_value = mock_instance
+        
+        # Setup common mock responses
+        mock_instance.get_system_status.return_value = {"status": "operational", "version": "1.0.0"}
+        
+        yield mock_instance
+
+@pytest.fixture
+def test_client(mock_ai_council):
```

Ruff (0.15.1)

[warning] Unused function argument: `mock_ai_council`

(ARG001)